lib/Client.test.js

```diff
@@ -185,7 +185,7 @@ describe("Client", () => {
         jest.advanceTimersByTime(0.5 * customClientOptions.interval);
 
         expect(pollSpy).toHaveBeenCalledTimes(3);
-        expect(engineService.fetchAndLock).toHaveBeenCalledTimes(2);
+        expect(engineService.fetchAndLock).toHaveBeenCalledTimes(1);
```

Comment:
I have no idea, why this change was needed. Actually, on my local environment, with Node  v14.15.4 even more counters were wrong. I assume reason being removal of  `json()` from got mock.